src/Azs.ContainerRegistry.Admin/examples/Set-AzsContainerRegistryQuota.md

```diff
@@ -0,0 +1,15 @@
+### Example 1: Set Azs ContainerRegistry Quota
+```powershell
+PS C:\> Set-AzsContainerRegistryQuota -QuotaName testquota -CapacityPerRegistryInGib 30 -NumberOfRegistry 30
```

Comment:
rename parameter to CapacityPerRegistryInGB

Reply:
We are following the same pattern as Azure do measuring in GiB instead of Gb